static/app/components/u2f/webAuthnHelper.tsx

```diff
@@ -0,0 +1,36 @@
+export function base64urlToBuffer(baseurl64String: string): ArrayBuffer {
```

Comment:
did you write this code? or is it from somwhere?

Reply:
this is from somewhere, will add the credits for the origin